contracts/RiskFund/RiskFund.sol

```diff
@@ -128,6 +137,46 @@ contract RiskFund is OwnableUpgradeable, ExponentialNoError {
         minAmountToConvert = _minAmountToConvert;
     }
 
+    /**
+     * @dev Update the reserve of the asset for the specific pool after transferring to risk fund.
+     * @param comptroller  Comptroller address(pool).
+     * @param asset Asset address.
+     */
+    function updateAssetsState(address comptroller, address asset) external {
+        require(
+            comptroller != address(0),
+            "Liquidated shares Reserves: Comptroller address invalid"
+        );
```

Comment:
should we also check that comptroller address actually implements Comptroller interface by checking `comptroller.isComptroller` ?

Reply:
Done.